components/server/src/storage/gcloud-storage-client.ts

```diff
@@ -39,7 +39,17 @@ export class GCloudStorageClient implements StorageClient {
     }
 
     async deleteBucket(bucketName: string): Promise<any> {
-        const { response, err } = await this.try(() => this.storage.bucket(bucketName).delete().then(responses => responses[0]));
+        const bucket = this.storage.bucket(bucketName);
```

Comment:
while we're at it, maybe an info log statement would be handy here, especially if things go wrong (`// ignore` down below).

Reply:
sounds good